canvas_account_admin_tools/templates/canvas_account_admin_tools/dashboard_account.html

```diff
@@ -56,24 +51,13 @@ <h4 class="list-group-item-heading">Courses in this Account</h4>
         </div>
       {% endif %}
 
-      {% if course_info %}
         <div class="list-group col-md-4">
           <a type="button" class="btn btn-primary list-group-item active" href="{% url 'course_info:index' %}" class="list-group-item active">
               <h4 class="list-group-item-heading">Find Course Info</h4>
               <p class="list-group-item-text">Search for course information <br>&nbsp;</p>
           </a>
         </div>
-      {% endif %}
 
-    {% endif %}
 
-    {% if not has_manage_courses and not has_manage_account %}
-    <div class="col-md-6">
-        <div class="panel panel-default">
-            <div class="panel-body">
-                <p>There are no administration tools installed in this account.</p>
-            </div>
-        </div>
-    </div>
-    {% endif %}
+
```

Comment:
They should get permission denied message, we can test and add ticket if that's not desired behavior.